language/tuning/distilling_step_by_step/Distillation.ipynb

```diff
@@ -0,0 +1,606 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    def tokenize_function(examples):
</pre><p>nit: Can you add parameter and return value types throughout?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Done. I have added in most places, but had to omit while overriding HuggingFace classes as transformers does not have complete typing yet</p>